### ISSUE TITLE
Add documentation for tartiflette-starlette

### ISSUE THREAD

**Issue by florimondmanca** [dailymotion/tartiflette#230]
## Request a feature

* A part of the project could be improved

The ASGI adapter for Tartiflette is being developer at https://github.com/tartiflette/tartiflette-starlette.

We discussed adding documentation for `tartiflette-starlette` alongside `tartiflette-aiohttp` — see https://github.com/tartiflette/tartiflette-starlette/issues/19#issuecomment-492319867.

A pre-requisite to this is to achieve feature parity between both packages (see https://github.com/tartiflette/tartiflette-starlette/issues/20).

This issue is mostly here to track this task somewhere. :-) I will probably start working on it once feature parity has been rolled out.

**Comment by tsunammis** [comment]
Thank you @florimondmanca I will have a look at it next week.

**Comment by tomchristie** [comment]
Thought: Might want to just call it `tartiflette-asgi` or something, and document it as mountable under Starlette, FastAPI, Responder, Bocadillo, Quart, or whatever else. (Possibly also the latest versions of Sanic, but I've not looked into that in any detail)

**Comment by florimondmanca** [comment]
Fair point @tomchristie. tartiflette-asgi was the initial name, but seeing how much confusion the current seems to bring wrt to the Starlette dependency (“My framework is not based on Starlette so I can’t use this, right?”), I’d also be keen to revert.

(As for Sanic, someone’s trying out Sanic + tartiflette-starlette right now, but from at first glance it seems that Sanic misses the ASGI sub-mounting that allows integrating with other frameworks.)

@tsunnamis To catch up, has there been any progress I haven’t seen on documenting tartiflette-starlette in the official docs?

**Comment by AutumnalDream** [comment]
What's the status and is there anything I can help with on this?

**Comment by florimondmanca** [comment]
@AutumnalDream I don't think there has been progress since the opening of this issue. Hints on how to contribute docs are [here](https://github.com/tartiflette/tartiflette#how-to-contribute-to-the-documentation). I think the first step would be to rephrase the highlight on the docs first page, e.g.:

```markdown
### aiohttp / ASGI

In addition to the Tartiflette engine,
`tartiflette-aiohttp` and `tartiflette-starlette` allow you to
serve your GraphQL API over HTTP.
```
